### ISSUE TITLE
Completion logic?

### ISSUE THREAD

**Issue by bmabey** [alexarchambault/jupyter-scala#19]
Great library! Your bootstrapping logic to setup a new kernel is quite nice.

Some of the other Scala kernel's I've used had completion logic so the Notebook will autocomplete method calls for you. I'm not seeing that with my version of `jupyter-scala` (using the 2.10.5 version and the latest SNAPSHOT on maven). Has that not been implemented yet or is it a problem on my side?

**Comment by alexarchambault** [comment]
Thanks.

Tab-completion should work, just checked with the [2.10 tar.xz link](https://oss.sonatype.org/content/repositories/snapshots/com/github/alexarchambault/jupyter/jupyter-scala-cli_2.10.5/0.2.0-SNAPSHOT/jupyter-scala_2.10.5-0.2.0-SNAPSHOT.tar.xz) from the README, like
![completion](https://cloud.githubusercontent.com/assets/7063723/8376065/612e7d4c-1c03-11e5-8a3a-80eb94b1ad52.PNG)

Did you double-check the link you're using? There are deprecated archives hanging around on sonatype, only the ones pointed at by the README should be used.

**Comment by bmabey** [comment]
I have verified that I'm using the correct jars. (I downloaded the zip version of the artifacts but I just confirmed the jar contents are the same using md5).

Taking a look again I can do class level tab-completion as illustrated in your notebook. However, I can not do instance level tab completion. With the Python kernel I can:
![screen shot 2015-06-26 at 4 23 32 pm](https://cloud.githubusercontent.com/assets/1837/8388476/bb8f5222-1c1f-11e5-9cc1-5a913f170be9.png)

With my Scala 2.10 kernel I can't:

![screen shot 2015-06-26 at 4 25 09 pm](https://cloud.githubusercontent.com/assets/1837/8388500/f5d24516-1c1f-11e5-9cf5-12a0d3d3f1bd.png)
